.github/workflows/sync_from_fungus.yml

```diff
@@ -0,0 +1,117 @@
+      GIT_AUTHOR_EMAIL: ${{ github.repository_owner }}@users.noreply.github.com
+      REPO_TEMPLATE: rust-lang/rust 
+      THIS_REPO: ${{ github.repository }}
+    
+    jobs:
+      sync-from-template:
+        # Do not run on the template repository itself
+        if: github.repository != 'rust-lang/rust'
+        name: Sync changes from rust-lang/rust 
+        runs-on: ubuntu-latest
+        continue-on-error: true
+    
+        steps:
+          # Clone the template repository
+          - name: Check out template repository
+            uses: actions/checkout@v2
+            with:
+              repository: ${{ env.REPO_TEMPLATE }}
+              token: ${{ github.token }}
+              path: ${{ env.REPO_TEMPLATE }}
+    
+          # Clone the target repository. Check out a branch
+          - name: Check out ${{ github.repository }}
+            uses: actions/checkout@v2
+            with:
+              repository: ${{ github.repository }}
+              token: ${{ github.token }}
+              path: ${{ github.repository }}
+          - name: Create branch in ${{ env.THIS_REPO }}
+            run: |
+              git -C "${THIS_REPO}" fetch origin "${HEAD_BRANCH}" || true
+              git -C "${THIS_REPO}" branch -a
+              git -C "${THIS_REPO}" checkout -B "${HEAD_BRANCH}" \
+                "remotes/origin/${HEAD_BRANCH}" || \
+              git -C "${THIS_REPO}" checkout -b "${HEAD_BRANCH}"
+    
+          # Copy files from the template onto the target clone
+          - name: Copy template contents
+            run: |
+              _files="$(find ${REPO_TEMPLATE} \
+                ! -path "*/.git/*" \
+                ! -path "*/.github/workflows/*" \
+                ! -name ".gitignore" \
+                ! -name "README.md" \
+                -type f \
+                -print)"
+              for _file in ${_files}; do
+                _src="${_file}"
+                _dst="${THIS_REPO}/${_file#${REPO_TEMPLATE}/}"
+                # TODO: Find a more robust / elegant way to get this :point_down:
+                _dst="${_dst%/*}/"
+                mkdir -p "${_dst}"
+                echo "INFO: Copy '${_src}' to '${_dst}'."
+                cp "${_src}" "${_dst}"
+              done
+              git -C "${THIS_REPO}" diff
+    
+          # Commit changes, if there are any
+          - name: Commit changes, if any
+            run: |
+              git -C ${THIS_REPO} config user.name "${GIT_AUTHOR_NAME}"
+              git -C ${THIS_REPO} config \
+                user.email "${GIT_AUTHOR_EMAIL}"
+              git -C ${THIS_REPO} add .
+              git -C ${THIS_REPO} commit \
+                -m "Sync from template@${{ github.sha }}"
+    
+          # Push the branch to the target repository
+          - name: Push topic branch
+            run: git -C ${THIS_REPO} push -u origin "${HEAD_BRANCH}"
+    
+          # Create a pull request in the target repository
+          - name: Create pull request
```

Comment:
this would still need to be manually merged into master, right?

Reply:
Yes! The idea was that the changes can be reviewed first.
This could be simplified if there is a need to commit directly to master, for example, but I wouldn't recommend it because you don't know what has changed. 

Even if you don't care what has changed, you can merge without a review.
It also helps to undo merge commits if you find out later that something is broken.